lib/rubocop/cop/salsify/i18n.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module I18n
+      module RailsI18n
+        class DecorateString < Cop
+          # Original: https://github.com/puppetlabs/rubocop-i18n/blob/v3.0.0/lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb#L138
+          # This patch adds the parent_is_logger? check
+          def check_for_parent_decorator(node)
```

Comment:
I think you can simplify this approach and make it easier for future maintainability by using `Module.prepend`.
Something like

```
Module MyPatch
  def check_for_parent_decorator(node)
    return if parent_is_logger?(node.parent)

    super(node)
  end

          private

          def parent_is_logger?(parent)
            parent.respond_to?(:method_name) && [:debug, :info, :warn, :error].include?(parent.method_name)
          end

end

RuboCop::Cop::I18n::RailsI18n::DecorateString.prepend(MyPatch)
```

Reply:
Ah yes, I really should move past ruby 1.9 😄 

Will fix!